pkg/cmd/roachtest/allocator.go

```diff
@@ -351,7 +351,10 @@ FROM crdb_internal.kv_store_status
 			}
 			t.l.Printf("%d unavailable, %d under-replicated ranges\n", unavailable, underReplicated)
 			if unavailable != 0 {
-				t.Fatalf("%d unavailable ranges", unavailable)
+				if timeutil.Since(start) >= 30*time.Second {
```

Comment:
Add a comment here that a freshly started cluster might show some ranges as unavailable even though they're not.

Reply:
Done.